### PR TITLE
Add loadflow an security analysis server to gateway

### PR DESCRIPTION
**Pull request by geofjamg**
Signed-off-by: Geoffroy Jamgotchian <geoffroy.jamgotchian@rte-france.com>

### CODE REVIEW

**Review by bendaamerahm — APPROVED** [review]

File: src/main/java/org/gridsuite/gateway/ServiceURIsConfig.java [hunk 1]
```diff
@@ -87,4 +87,9 @@ public class ServiceURIsConfig {
     @Value("${gridsuite.services.sensitivity-analysis-server.base-uri:http://sensitivity-analysis-server/}")
     String sensitivityAnalysisServerBaseUri;
 
+    @Value("${backing-services.loadflow-server.base-uri:http://loadflow-server/}")
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/gateway/ServiceURIsConfig.java:90]: gridsuite.services.loadflow-server

File: src/main/java/org/gridsuite/gateway/ServiceURIsConfig.java [hunk 2]
```diff
@@ -87,4 +87,9 @@ public class ServiceURIsConfig {
...
+    String loadFlowServerBaseUri;
+
+    @Value("${backing-services.security-analysis-server.base-uri:http://security-analysis-server/}")
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/gateway/ServiceURIsConfig.java:93]: gridsuite.services.security-analysis-server

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/gateway/ServiceURIsConfig.java (hunk 1 above)
Comment (geofjamg) [reply src/main/java/org/gridsuite/gateway/ServiceURIsConfig.java]: Done

File: src/main/java/org/gridsuite/gateway/ServiceURIsConfig.java (hunk 2 above)
Comment (geofjamg) [reply src/main/java/org/gridsuite/gateway/ServiceURIsConfig.java]: Done

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**